syntax = "proto3";
package syft.core.auth;

import "proto/core/common/common_object.proto";
import "google/protobuf/empty.proto";

message SignedMessage {
  string obj_type = 1;
  bytes signature = 2;
  bytes verify_key = 3;
  bytes message = 4;
}

message VerifyKey { bytes verify_key = 1; }
message VerifyAll { google.protobuf.Empty all = 1; }